static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-articles-viewed.js

```diff
@@ -54,7 +55,8 @@ export const articlesViewed: EpicABTest = makeEpicABTest({
     audience: 1,
     audienceOffset: 0,
 
-    canRun: () => articleViewCount >= minArticleViews,
+    canRun: () =>
+        articleViewCount >= minArticleViews && countryNames[geolocation],
```

Comment:
not sure if this will pass the linter but I like parens:
```suggestion
        (articleViewCount >= minArticleViews) && countryNames[geolocation],
```